fili-core/src/main/java/com/yahoo/bard/webservice/web/JsonAndJsonApiResponseWriter.java

```diff
@@ -71,6 +78,35 @@ public void writeMetaObject(
 
         generator.writeObjectFieldStart("meta");
 
+        if (request instanceof DataApiRequest) {
```

Comment:
we should put this behavior in it's own method so it's easy to override.
We also should put tests on that method that test the various serializations (with/without type, with/without subtype and with/without typeMetadata)